src/content/es/blog/sora-card-vs-others.md

```diff
@@ -16,24 +16,24 @@ La Tarjeta SORA tiene como objetivo proporcionar una experiencia de estilo banca
+| Who Possesses My Assets?        | Binance (Crypto)                                 | Crypto.com (Crypto) + IBAN provider (Fiat)          | Choise (Crypto) + IBAN provider(Fiat)                     | Revolut (Crypto + Fiat)                       | YOU (Crypto) + IBAN provider (Fiat)                                                                              |
+| Who Possesses My Personal Data? | Binance + IBAN provider                          | Crypto.com + IBAN provider                          | Choise + IBAN provider                                    | Revolut                                       | YOU + IBAN provider not SORA                                                                                     |
+| Exchange Type                   | Centralized                                      | Centralized                                         | Centralized                                                   | Centralized                                   | Decentralized (Crypto↔Crypto) + Centralized (Crypto↔EUR)                                                            |
+| Cash Withdrawal Limit           | EUR 290 per day                                  | EUR 5000-10000\* per day <br>\*Depends on card        | EUR 500 per day / EUR 2,500 per month                         | EUR 3,000 per day                             | EUR 5,000\* per day |
+| Spending Limit                  | EUR 8,700 per day                                | EUR 25,000 per day                                  | EUR 10,000 per month                                          | No spending limit / Max. 50 transfers per 24h | EUR 25,000\* per day                                |
+| Top-Up Limit                    | Auto-conversion spending limit EUR 8,700 per day | EUR 25,000 per month                                | EUR 5,000 per day 60,000 per month                            | No crypto top-up enabled                      | EUR 25,000 per day\*                                |
 | Top-Up Options                  | Crypto — Autoconversion                          | Crypto — top up for spending only, no fiat off ramp | Crypto + Fiat bank transfer                                   | Fiat: bank transfer, credit card, debit card  | Crypto (XOR) + fiat: bank transfer, credit card, debit card                                                      |
 | IBAN                            | No                                               | Yes                                                 | Yes                                                           | Yes                                           | Yes                                                                                                              |
 
-Vea la tabla de comparación en formato PDF: [/pub/sora-card-table-alt.pdf](https://soracard.com/pub/sora-card-table-alt.pdf)
+Vea la tabla de comparación en formato PDF: [/pub/sora-card-table-alt.pdf](/pub/sora-card-table-alt.pdf)
```

Comment:
@tonyfromsora Sorry, it looks like `NuxtLink` doesn't understands that this relative link points to a static asset, which causing 404, you can see it [here](https://soracard-blog-9zbib2qmk-sora-card-blogs-projects.vercel.app/blog/sora-card-vs-others), so can you please return direct links. It will work on production.

Thank you

Reply:
@lsvs it seems to me it's done!